plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -458,4 +560,11 @@ public void setNull(PreparedStatement statement, int index)
                 .findOne()
                 .flatMap(dataCompression -> Enums.getIfPresent(DataCompression.class, dataCompression).toJavaUtil());
     }
+
+    private static Calendar newUtcCalendar()
+    {
+        Calendar calendar = new GregorianCalendar(UTC_TIME_ZONE, ENGLISH);
+        calendar.setTime(new Date(0));
```

Comment:
this is redundant. if the results depended on the calendar timestamp, it would be easy to spot too

and if you were to set calendar state to a very well known state, you probably would need to call some other methods too (?)

Reply:
Done, and all tests pass.